Mathlib/Order/Monotone/Basic.lean

```diff
@@ -1068,6 +1068,18 @@ theorem Monotone.prod_map (hf : Monotone f) (hg : Monotone g) : Monotone (Prod.m
 theorem Antitone.prod_map (hf : Antitone f) (hg : Antitone g) : Antitone (Prod.map f g) :=
   fun _ _ h ↦ ⟨hf h.1, hg h.2⟩
 
+lemma monotone_separately_iff_jointly {h : α × β → γ} :
+    (∀ a, Monotone (h ∘ Prod.mk a)) ∧ (∀ b, Monotone (fun a => h (a, b))) ↔ Monotone h where
```

Comment:
I'd guess this makes a bit more sense in the reverse direction, that is giving a simpler way of proving a function from a product is monotone by showing certain restrictions of it are monotone

Reply:
Done.